parlai/core/torch_agent.py

```diff
@@ -317,6 +341,18 @@ def _update_vecs(self, text):
                 self.history_vecs.pop(0)
         self.history_vecs.append(self.parse(text))
 
+    def _update_timesteps(self, timestep):
+        if self.size > 0:
+            while len(self.history_timesteps) >= self.size:
+                self.history_timesteps.pop(0)
+        self.history_timesteps.append(timestep)
+
+    def _update_timestep_vecs(self, timestep):
```

Comment:
I don't see any situation that you only call the `_update_timestep_vecs` function. And it looks a lot like the `_update_timesteps` function. Is it possible to merge these two into one function? They seem to be doing very similar things and they are called together.